### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Name property is empty for edit in "Create App Service dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1398]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Publish as Azure web app.
4. "Deploy web app" dialog will open. 
5. In "Deploy web app" dialog , navigate to and select "Create" button.
6. In "Create App Service" dialog , tab to and select "Create New" radio button.
7. Observe the name property for the create new edit box using inspect.

**Actual Result:**
Name property is empty for the edit box

**Narrator behaviour:**
editing asp-180129104404

**NVDA behaviour:**
edit
 asp-180129104404

**Expected Result:**
Name property for the edit should be "Create New".

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![create app service-app service plan-edit](https://user-images.githubusercontent.com/22956130/35684245-503e5c74-078c-11e8-953c-04717dc12140.jpg)
![create app service-app service plan-edit-nvda](https://user-images.githubusercontent.com/22956130/35684254-53a1a826-078c-11e8-942b-2df8326f6f34.jpg)

**Comment by jdneo** [comment]
fixed

![untitled](https://user-images.githubusercontent.com/6193897/38715763-aad18f2a-3f0f-11e8-8a7e-cc56de775f42.png)

**Comment by Eskibear** [comment]
shipped in v3.6.0

**Comment by WiproPSAccTest** [comment]
Verified the issue in V.3.6.0 .Issue got fixed.
Please add closed tag.
![1398-fixed](https://user-images.githubusercontent.com/22956130/39761428-b2691c18-52f5-11e8-98b6-e8fccff0d3fb.jpg)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749839
